exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/JSONRecordReader.java

```diff
@@ -189,39 +194,37 @@ private long currentRecordNumberInFile() {
   public int next() {
     writer.allocate();
     writer.reset();
-
     recordCount = 0;
     ReadState write = null;
 //    Stopwatch p = new Stopwatch().start();
-    try{
-      outside: while(recordCount < DEFAULT_ROWS_PER_BATCH) {
-        writer.setPosition(recordCount);
-        write = jsonReader.write(writer);
-
-        if(write == ReadState.WRITE_SUCCEED) {
+   // try
+   // {
+      outside: while(recordCount < DEFAULT_ROWS_PER_BATCH){
+      try{
+            writer.setPosition(recordCount);
+            write = jsonReader.write(writer);
+            if(write == ReadState.WRITE_SUCCEED) {
 //          logger.debug("Wrote record.");
-          recordCount++;
-        }else{
+              recordCount++;
+            }else{
 //          logger.debug("Exiting.");
-          break outside;
-        }
-
+              break outside;
+            }
       }
-
-      jsonReader.ensureAtLeastOneField(writer);
-
+      catch(Exception ex)
+      {
+           ++parseErrorCount;
```

Comment:
the indentations seem to be off here as well as other places.. can you make sure the indentations match the rest of the code ?